packages/Confirmation/src/Confirmation.js

```diff
@@ -10,50 +10,52 @@ import TriggerButton from "./components/TriggerButton";
 import { confirmStyles, confirmBodyStyles, confirmFooterStyles } from "./Confirmation.styles";
 
 const propTypes = {
+  body: PropTypes.node,
```

Comment:
I started making changes to this `Confirmation.js` file, then realized it was unnecessary.  Rather than revert some code tidy up, I left it in here, but none of it introduces any changes.